.github/workflows/main.yml

```diff
@@ -0,0 +1,41 @@
+name: CI
+
+on: [push]
+
+jobs:
+  images:
+    name: Images
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Example image
+        run: ./kas-docker build kas-iot2050-example.yml
+      - name: Upload example image
+        uses: actions/upload-artifact@v2
+        with:
+          name: iot2050-example-image
+          path: build/tmp/deploy/images/iot2050/iot2050-image-example-isar-iot2050.wic.img*
```

Comment:
How many disk spaces do we have?

Reply:
No idea. I didn't find notes on that on github yet. Maybe you can try to search as well. But if there is a limit: There is also no hint how much you used already.

Reply:
> No idea. I didn't find notes on that on github yet. Maybe you can try to search as well. But if there is a limit: There is also no hint how much you used already.

I tried to search it but didn't find anything, I am wondering if we are using the organization storage space...